### PR TITLE
Add new field for AKS

### PR DESCRIPTION
**Pull request by n313893254**
Proposed changes
======
1. Change filed name:  osDiskSizeGb => agentOsdiskSize
2. Add fields 
```
enableMonitoring  
logAnalyticsWorkspaceResourceGroup
 logAnalyticsWorkspace
// according backend implement, UI won’t show logAnalyticsWorkspaceResourceGroup and 
// logAnalyticsWorkspace now.

enableHttpApplicationRouting
tags
networkPolicy
```
3. Support AKS editable in UI.

Types of changes
======

- New feature (non-breaking change which adds functionality)

Linked Issues
======

https://github.com/rancher/rancher/issues/17262

 

Further comments
======

### CODE REVIEW

**Comment by cjellick** [comment]
backend is in

File: translations/en-us.yaml [hunk 1]
```diff
@@ -2245,6 +2245,39 @@ clusterNew:
       placeholder: Your Client Secret
     region:
       label: Region
+    aadClientAppId:
+      label: Client ID
+      placeholder: Your Client ID of Azure Active Directory.
```
Comment (westlywright) [inline translations/en-us.yaml]: Remove the periods in all the new placeholders, some have them and some don't. We should be consistent and for place holders we typically don't use periods.

File: translations/zh-hans.yaml [hunk 2]
```diff
@@ -2167,6 +2167,39 @@ clusterNew:
       placeholder: 客户端秘钥
     region:
       label: 区域
+    aadClientAppId:
+      label: Client ID
+      placeholder: Your Client ID of Azure Active Directory.
```
Comment (westlywright) [inline translations/zh-hans.yaml]: Remove the periods in all the new placeholders, some have them and some don't. We should be consistent and for place holders we typically don't use periods.

File: lib/shared/addon/components/cluster-driver/driver-azureaks/template.hbs [hunk 3]
```diff
@@ -136,15 +130,74 @@
...
+                  type="text"
+                  value=config.masterDnsPrefix
+              }}
+            {{/input-or-display}}
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col span-6">
+            <label class="acc-label">{{t 'clusterNew.azureaks.enableHttpApplicationRouting.label'}}</label>
```
Comment (westlywright) [inline lib/shared/addon/components/cluster-driver/driver-azureaks/template.hbs]: We're mixing single and double quotes in the same html file. We dont have a template linter yet (coming in the next ember upgrade) but when we do the standard is double qoutes in HBS files.

If you are working on an HBS file I am okay if you do a global (file global) find and replace of single quotes to double quotes.

**Comment by n313893254** [comment]
@westlywright Done, thanks for your review.

**Pull request merged**